cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -1,164 +1,99 @@
 
-@pytest.mark.uncollectif(lambda: version.current_version() >= '5.7')
-@pytest.fixture(scope="module")
-def test_instance(setup_provider_modscope, request, delete_instances_fin, provider, vm_name):
-    """ Fixture to provision instance on the provider
-    """
-    instance = Instance.factory(vm_name, provider)
-    if not provider.mgmt.does_vm_exist(vm_name):
-        delete_instances_fin[provider.key] = instance
-        instance.create_on_provider(allow_skip="default")
+    if not a_provider.mgmt.does_vm_exist(instance.name):
+        logger.info("deploying %s on provider %s", instance.name, a_provider.key)
+        instance.create_on_provider(allow_skip="default", find_in_cfme=True)
     return instance
 
 
 @pytest.fixture(scope="module")
-def gen_events(setup_provider_modscope, delete_fx_provider_event, provider, test_instance):
+def gen_events(test_instance):
     logger.debug('Starting, stopping VM')
-    mgmt = provider.mgmt
+    mgmt = test_instance.provider.mgmt
     mgmt.stop_vm(test_instance.name)
     mgmt.start_vm(test_instance.name)
-    provider.refresh_provider_relationships()
 
 
-def count_events(instance_name, nav_step):
-    try:
-        nav_step()
-    except ToolbarOptionGreyedOrUnavailable:
-        return 0
-    events = []
-    for event in jstimelines.events():
-        data = event.block_info()
-        if instance_name in data.values():
-            events.append(event)
-            if len(events) > 0:
-                return len(events)
-    return 0
-
-
-def db_event(db, provider):
-    # Get event count from the DB
-    logger.info("Getting event count from the DB for provider name %s", provider.name)
-    ems = db['ext_management_systems']
-    ems_events = db['event_streams']
-    with db.transaction:
-        providers = (
-            db.session.query(ems_events.id)
-            .join(ems, ems_events.ems_id == ems.id)
-            .filter(ems.name == provider.name)
-        )
-        query = db.session.query(ems_events).filter(ems_events.id.in_(providers.subquery()))
-        event_count = query.count()
-    return event_count
-
-
-@pytest.mark.uncollectif(lambda: version.current_version() >= '5.7')
-@pytest.mark.meta(blockers=[BZ(1201923, unblock=lambda provider: provider.type != 'ec2',
-                               forced_streams=['5.6']),
-                            BZ(1390572, unblock=lambda provider: provider.type != 'azure',
-                               forced_streams=['5.6'])])
-def test_provider_event(setup_provider, provider, gen_events, test_instance):
+def count_events(target, vm):
+    timelines_view = navigate_to(target, 'Timelines')
+    timelines_view.filter.time_position.select_by_visible_text('centered')
+    timelines_view.filter.apply.click()
+    found_events = []
+    for evt in timelines_view.chart.get_events():
+        # BZ(1428797)
+        if not hasattr(evt, 'source_vm'):
+            logger.warn("event {evt} doesn't have source_vm field. Probably issue".format(evt=evt))
```

Comment:
Well, IIRC, you don't use `.format()` with the logger methods, you use %s/%r and pass the values as additional parameters after the log message string.